src/main/java/org/opensearch/knn/index/codec/KNN80Codec/KNN80DocValuesConsumer.java

```diff
@@ -129,6 +132,19 @@ public void addKNNBinaryField(FieldInfo field, DocValuesProducer valuesProducer)
         writeFooter(indexPath, engineFileName);
     }
 
+    private boolean isLuceneEngine(FieldInfo field) {
+        final KNNEngine knnEngine;
```

Comment:
Lets add debug logger to know which version of KNN is chosen like Lucene or Native

Reply:
agree, will update accordingly